.github/workflows/ci.yaml

```diff
@@ -26,6 +26,12 @@ jobs:
         command: lint
         config: .github/ct.yaml
 
+  lint-appVersion:
```

Comment:
`lint-appVersion` is executed despite if jsonnet dir is not changed.   
Use [jobs.<job_id>.if](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif)  to prevent a job from running unless a condition is met.

Reply:
Check if jsonnet dir changed is [here](https://github.com/dNationCloud/kubernetes-monitoring/blob/fix-ci/.github/workflows/ci.yaml#61). I didn't find any build-in way to execute job only if specifc dir changed. 
I could move [this step](https://github.com/dNationCloud/kubernetes-monitoring/blob/fix-ci/.github/workflows/ci.yaml#60) to separate job, set output in it and then check it with ` jobs.<job_id>.if` in another job, but I dont kwno if it is a better solution.

Reply:
My point is that following steps are executed despite the condition is no met:
- `Get new helm chart appVersion`
- `Checkout target branch of pull request`
- `Get old helm chart appVersion`

I see two possible solutions:
1.  Add condition `if: steps.changes.outputs.jsonnet == 'true'` to all above mentioned steps
2. Move `Check if jsonnet folder changed` to the previous job and then check it with jobs.<job_id>.if in another job (as you describe above)

Reply:
Because It is not a blocker, but only nice to have improvement I created follow up [issue](https://github.com/dNationCloud/kubernetes-monitoring/issues/46). This PR will be merged as hot fix.